src/shaders/particles.glsl

```diff
@@ -79,6 +79,12 @@ vec2 update(vec2 pos) {
     vec2 velocity = mix(u_wind_min, u_wind_max, lookup_wind(wind_tex_pos));
     float speed_t = length(velocity) / length(u_wind_max);
 
+    // lookup wind mask in b channel
+    float b = texture2D(u_wind_middle_center, wind_tex_pos).b;
+    // check if mask is > 0.5
```

Comment:
comment doesn't match code.
why this 0.5 or 0.1?

Reply:
fixed